modules/cluster-logging-updating-logging-46.adoc

```diff
@@ -0,0 +1,324 @@
+green  open   project.nw.6233ad57-aff0-4d5a-976f-370636f47b11.2020.06.30            dtc6J-nLSCC59EygeV41RQ   3 1           10            0          0              0
+green  open   infra-000001                                                          eH53BQ-bSxSWR5xYZB6lVg   3 1       341800            0        443            220
+green  open   .kibana-6                                                             RVp7TemSSemGJcsSUmuf3A   1 1            4            0          0              0
+green  open   infra-000011                                                          J7XWBauWSTe0jnzX02fU6A   3 1       226100            0        293            146
+green  open   app-000001                                                            axSAFfONQDmKwatkjPXdtw   3 1       103186            0        126             57
+green  open   infra-000016                                                          m9c1iRLtStWSF1GopaRyCg   3 1        13685            0         19              9
+green  open   infra-000002                                                          Hz6WvINtTvKcQzw-ewmbYg   3 1       228994            0        296            148
+green  open   project.qt.2c05acbd-bc12-4275-91ab-84d180b53505.2020.06.30            MUm3eFJjSPKQOJWoHskKqw   3 1        12262            0         14              7
+green  open   infra-000013                                                          KR9mMFUpQl-jraYtanyIGw   3 1       228166            0        298            148
+green  open   audit-000001                                                          eERqLdLmQOiQDFES1LBATQ   3 1            0            0          0              0
+----
+====
+
+.. Verify that the log collector is updated to 4.6:
++
+[source,terminal]
+----
+$ oc get ds fluentd -o json | grep fluentd-init
+----
++
+You should see a `fluentd-init` container.
++
+.Example output
+[source,terminal]
+----
+"containerName": "fluentd-init"
+----
+
+.. Verify that the log visualizer is updated to 4.6 using the Kibana CRD:
++
+[source,terminal]
+----
+$ oc get kibana kibana -o json
+----
++
+You should see a Kibana pod with the `ready` status:
++
+.Example output with a ready Kibana pod
+[%collapsible]
+====
+[source,json]
+----
+[
+{
+"clusterCondition": {
+"kibana-5fdd766ffd-nb2jj": [
+{
+"lastTransitionTime": "2020-06-30T14:11:07Z",
+"reason": "ContainerCreating",
+"status": "True",
+"type": ""
+},
+{
+"lastTransitionTime": "2020-06-30T14:11:07Z",
+"reason": "ContainerCreating",
+"status": "True",
+"type": ""
+}
+]
+},
+"deployment": "kibana",
+"pods": {
+"failed": [],
+"notReady": []
+"ready": []
+},
+"replicaSets": [
+"kibana-5fdd766ffd"
+],
+"replicas": 1
+}
+]
+----
+====
+
+.. Verify the Curator is updated to 4.6:
++
+[source,terminal]
+----
+$ oc get cronjob -o name
+----
++
+.Example output
+[source,terminal]
+----
```

Comment:
$  oc get cronjob -o name
cronjob.batch/curator
cronjob.batch/elasticsearch-im-app
cronjob.batch/elasticsearch-im-audit
cronjob.batch/elasticsearch-im-infra

Reply:
Fixed. 

@anpingli - Should I keep this sentence? "You should see the `elasticsearch-delete-\*` and `elasticsearch-rollover-*` indices."

Reply:
@anpingli  Rick Rackow confirmed the answer to my question, and I have added a clarification that it takes approximately 30 minutes after installation /upgrade to see these indices - per Rick. Please review and verify this PR now if you can. Thanks.